compling without FFTW is broken

Either with `./configure --without-fftw` or `cmake -DBOUT_USE_FFTW=OFF`
```
fft_fftw.cxx: In function ‘unsigned int bout::fft::get_measurement_flag(FFT_MEASUREMENT_FLAG)’:
fft_fftw.cxx:74:14: error: ‘FFTW_ESTIMATE’ was not declared in this scope
   74 |       return FFTW_ESTIMATE;
      |              ^~~~~~~~~~~~~
fft_fftw.cxx:76:14: error: ‘FFTW_MEASURE’ was not declared in this scope
   76 |       return FFTW_MEASURE;
      |              ^~~~~~~~~~~~
fft_fftw.cxx:78:14: error: ‘FFTW_EXHAUSTIVE’ was not declared in this scope
   78 |       return FFTW_EXHAUSTIVE;
      |              ^~~~~~~~~~~~~~~
```

--- a/src/invert/fft_fftw.cxx
+++ b/src/invert/fft_fftw.cxx
@@ -69,18 +69,20 @@ void fft_init(Options* options) {
 }
 
 #if BOUT_HAS_FFTW
+namespace {
 unsigned int get_measurement_flag(FFT_MEASUREMENT_FLAG fft_measurement_flag) {
   switch (fft_measurement_flag) {
-    case FFT_MEASUREMENT_FLAG::estimate:
-      return FFTW_ESTIMATE;
-    case FFT_MEASUREMENT_FLAG::measure:
-      return FFTW_MEASURE;
-    case FFT_MEASUREMENT_FLAG::exhaustive:
-      return FFTW_EXHAUSTIVE;
-    default:
-      throw BoutException("Error, unimplemented fft_measurement_flag");
+  case FFT_MEASUREMENT_FLAG::estimate:
+    return FFTW_ESTIMATE;
+  case FFT_MEASUREMENT_FLAG::measure:
+    return FFTW_MEASURE;
+  case FFT_MEASUREMENT_FLAG::exhaustive:
+    return FFTW_EXHAUSTIVE;
+  default:
+    throw BoutException("Error, unimplemented fft_measurement_flag");
   }
 }
+} // namespace
 #endif
 
 void fft_init(FFT_MEASUREMENT_FLAG fft_measurement_flag) {

--- a/src/invert/fft_fftw.cxx
+++ b/src/invert/fft_fftw.cxx
@@ -68,6 +68,7 @@ void fft_init(Options* options) {
   fft_init(fft_measurement_flag);
 }
 
+#if BOUT_HAS_FFTW
 unsigned int get_measurement_flag(FFT_MEASUREMENT_FLAG fft_measurement_flag) {
   switch (fft_measurement_flag) {
     case FFT_MEASUREMENT_FLAG::estimate:
@@ -80,6 +81,7 @@ unsigned int get_measurement_flag(FFT_MEASUREMENT_FLAG fft_measurement_flag) {
       throw BoutException("Error, unimplemented fft_measurement_flag");
   }
 }
+#endif
 
 void fft_init(FFT_MEASUREMENT_FLAG fft_measurement_flag) {
   bout::fft::fft_measurement_flag = fft_measurement_flag;

